driver/src/test/unit/com/mongodb/client/gridfs/GridFSBucketSpecification.groovy

```diff
@@ -432,28 +435,28 @@ class GridFSBucketSpecification extends Specification {
 
     def 'should create the expected GridFSFindIterable'() {
```

Comment:
This test should be re-named to something like "should execute the expected FindOperation when finding a file", as it's no longer asserting anything about the GridFSFindIterable.